Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser_description.md

```diff
@@ -1,14 +1,62 @@
 ## Application Authorization Flow
 
-To allow XSOAR to access Gmail, the user has to approve the Demisto app using an OAuth 2.0 authorization flow. Follow these steps to authorize the Demisto app in Gmail.
-
-1. Create and save an integration instance of the Gmail Single User integration. Do not fill in the *Auth Code* field, this will be obtained in the next steps.
-2. To obtain the **Auth Code** run the following command in the playground: ***!gmail-auth-link***. Access the link you receive to authenticate your Gmail account. 
-3. Complete the authentication process and copy the received code to the **Auth Code** configuration parameter of the integration instance. 
-4. Save the instance.
-5. To verify that authentication was configured correctly, run the ***!gmail-auth-test***.
-
-## Application Configuration Options
-See the [Integration Documentation](https://xsoar.pan.dev/docs/reference/integrations/gmail-single-user) for additional App configuration options:
-* **GSuite Admins:** How to trust the Demisto App.
-* **Bring your own App:** Use your own App instead of the default Demisto App. 
+* To allow Cortex XSOAR to access Gmail, you need to approve the Demisto App (for GSuite Admins) or create your own app (for all other account types).\
+    * To approve the Demisto app, follow the steps in 'GSuite Admins'.\
+    * To create your own app, follow the steps in 'All Account Types'.\
+* Once you have the app, follow the steps in 'Authorization Flow In Cortex XSOAR' to configure OAuth 2.0 authorization in Cortex XSOAR.
```

Comment:
Add here the link for `Authorization Flow In Cortex XSOAR`

Reply:
do you mean a link for the documentation in xsoar-pan-dev? e.g.: https://xsoar.pan.dev/docs/reference/integrations/gmail-single-user#application-authorization-flow

Reply:
An inline link to the right part in the document (same goes for all flows)